### PR TITLE
Color links in tweets and embedded media BGs

### LINKED ISSUE #44: Make tweets links and mentiones blue and make embedded tweets have a background
From: https://github.com/sindresorhus/anatine/pull/30#issuecomment-218670216

Like the official Twitter app:

### PR DESCRIPTION
**Pull request by paulmolluzzo**
This is a fixes #44.

I'm not sure I got the colors right, so if I need to change them just provide the hex/rgba.

screen shot 2016-05-14 at 8 57 38 pm

screen shot 2016-05-14 at 9 06 30 pm

Looks even nicer in the notifications view!

### CODE REVIEW

File: dark-mode.css [hunk 1]
```diff
@@ -62,6 +62,11 @@ html.dark-mode ._3EzK97M9 > span {
 	color: rgba(255, 255, 255, 0.75);
 }
 
+/* tweet links are spans in spans in spans */
+html.dark-mode ._3ZFqFXW8 ._3EzK97M9 > span > span{
+	color: #1da1f2 !important;
```
Comment (sindresorhus) [inline dark-mode.css]: I checked the app, the exact color is: `#2AA2EF`

File: dark-mode.css [hunk 2]
```diff
@@ -62,6 +62,11 @@ html.dark-mode ._3EzK97M9 > span {
 	color: rgba(255, 255, 255, 0.75);
 }
 
+/* tweet links are spans in spans in spans */
+html.dark-mode ._3ZFqFXW8 ._3EzK97M9 > span > span{
```
Comment (sindresorhus) [inline dark-mode.css]: `span{` => `span {`

Comment (paulmolluzzo) [inline dark-mode.css]: Fixed.

File: dark-mode.css (hunk 1 above)
Comment (paulmolluzzo) [inline dark-mode.css]: Fixed.

**Comment by sindresorhus** [comment]
🍰

**Pull request merged**